### PR TITLE
BezierEdgeRouter improvements

### PR DESCRIPTION
**Pull request by kaisalmen**
Implement review comments:
- Ensure the action is serializable
- Rename action and command
- Moved main execution code to command and removed unnecessary methods

Signed-off-by: Kai Salmen <kai.salmen@typefox.io>

### CODE REVIEW

File: packages/sprotty/src/features/routing/bezier-edge-router.ts [hunk 1]
```diff
@@ -317,79 +317,64 @@ export class BezierEdgeRouter extends AbstractEdgeRouter {
...
 };
 
-export enum BezierActionTask {
+export enum AddRemoveBezierSegmentActionTask {
     ADD,
     REMOVE
 }
 
-export class BezierCurveAction implements Action {
+export class AddRemoveBezierSegmentAction implements Action {
```
Comment (spoenemann) [inline packages/sprotty/src/features/routing/bezier-edge-router.ts:338]: Following my recent refactoring, I'd change this to an interface:
```typescript
export interface AddRemoveBezierSegmentAction extends Action {
    kind: typeof AddRemoveBezierSegmentAction.KIND
    targetId: string
    actionTask: 'add' | 'remove'
}
export namespace AddRemoveBezierSegmentAction {
    export const KIND = 'addRemoveBezierSegment';
    export function create(actionTask: 'add' | 'remove', targetId: string): AddRemoveBezierSegmentAction {
        return {
            kind: KIND,
            actionTask,
            targetId
        };
    }
}
```

File: packages/sprotty/src/features/routing/bezier-edge-router.ts [hunk 2]
```diff
@@ -317,79 +317,64 @@ export class BezierEdgeRouter extends AbstractEdgeRouter {
...
-                result.push(new BezierCurveAction(router, BezierActionTask.REMOVE, target.pointIndex, target.parent.id));
+                result.push(new AddRemoveBezierSegmentAction(AddRemoveBezierSegmentActionTask.REMOVE, target.id));
             }
         }
         return result;
     }
 };
 
-export enum BezierActionTask {
+export enum AddRemoveBezierSegmentActionTask {
```
Comment (spoenemann) [inline packages/sprotty/src/features/routing/bezier-edge-router.ts:333]: I'd use a simple string with constrained type: `'add' | 'remove'`

Comment (kaisalmen) [reply packages/sprotty/src/features/routing/bezier-edge-router.ts:333]: Yes, constraining a type like this makes enums "obsolete" in this case. 👍

File: packages/sprotty/src/features/routing/bezier-edge-router.ts (hunk 1 above)
Comment (kaisalmen) [reply packages/sprotty/src/features/routing/bezier-edge-router.ts:338]: I will adapt as you suggested. Do you want to refactor all actions implementations in the future?

Comment (spoenemann) [reply packages/sprotty/src/features/routing/bezier-edge-router.ts]: Yes, actions should always be defined as interfaces.

**Review by spoenemann — APPROVED** [review]
:+1:

**Pull request merged**